ansible/roles/content-snapshot-indexer/defaults/main.yml

```diff
@@ -0,0 +1,82 @@
+analytics_user: analytics
+analytics_group: analytics
+content_snapshot_path: "/mount/data/analytics/content-snapshot/"
+content_snapshot_jar_path: "{{content_snapshot_path}}jars/" # Add content snapshot jar file path.
+content_snapshot_jar_name: "adhoc-jobs-1.0.jar"
+content_snapshot_distribuction_name: "adhoc-jobs-1.0-distribution.tar.gz"
+spark_home: "/mount/data/analytics/spark-2.0.1-bin-hadoop2.7"
+redis:
+  host: "{{groups['redis'][0]}}" # Redis DP IP 
+  port: "6379"
+  user:
+    index: "4"
+    source_key: "id"
+  content:
+    index: "5"
+    source_key: "identifier"
+  dialcode:
+    index: "6"
+    source_key: "identifier"
+  connection:
+    idle_max: 20
+    idle_min: 10
+    min_evict_idle_seconds: 120
+    time_evict_run_second: 300
+    max: 20
+  max_pipeline_size: "1000"
+  key_expiry_seconds: 3600
+
+elastic_search:
+  host: "{{groups['core-es'][0]}}" # LP Elastic search IP Address
```

Comment:
The comment says LP ES, but group vars says core-es. Please check and update comment or code accordingly. In LP its called composite ES. In Core its called application ES